### PR TITLE
[15.0][MIG] stock_picking_assign_serial_final: Migration to version 15.0

### PR DESCRIPTION
**Pull request by ernesto-garcia-tecnativa**
@tecnativa TT38313

### CODE REVIEW

**Comment by ernesto-garcia-tecnativa** [comment]
@pedrobaeza @victoralmau please review

File: stock_picking_assign_serial_final/tests/common.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        cls.warehouse = cls.env.ref("stock.warehouse0")
+        cls.picking_type_in = cls.env.ref("stock.picking_type_in")
+        cls.supplier_location = cls.env.ref("stock.stock_location_suppliers")
+        cls.supplier = cls.env["res.partner"].create({"name": "Supplier - test"})
+
+    @classmethod
+    def _create_product(cls, tracking="serial", auto=True):
+        name = "{tracking} - {auto}".format(tracking=tracking, auto=auto)
+        return cls.env["product.product"].create(
+            {"name": name, "type": "product", "tracking": tracking}
```
Comment (victoralmau) [inline stock_picking_assign_serial_final/tests/common.py]:
```suggestion
            {"name": name, "detailed_type": "product", "tracking": tracking}
```

**Review by victoralmau — APPROVED** [review]
Code review OK.

**Comment by victoralmau** [comment]
Now finally, you just need to resolve the pre-commit errors.

**Comment by ernesto-garcia-tecnativa** [comment]
> Code review OK.

Thank you @victoralmau , I push changes, sorry I getting used to it

**Review by victoralmau — APPROVED** [review]
Code review OK.

**Review by rousseldenis — APPROVED** [review]
Code review

**Pull request merged**